src/main/xerus/monstercat/api/response/Release.kt

```diff
@@ -8,13 +8,12 @@ import xerus.ktutil.to
 private val logger = KotlinLogging.logger { }
 
 data class Release(
-	@Key("_id") override var id: String = "",
+	@Key override var id: String = "",
 	@Key var catalogId: String = "",
 	@Key var releaseDate: String = "",
 	@Key var type: String = "",
-	@Key var renderedArtists: String = "",
+	@Key("artistsTitle") var renderedArtists: String = "",
```

Comment:
```suggestion
	@Key var artistsTitle: String = "",
```
finally it's consistent, let us not focus on legacy support!

Reply:
Still needs a commit to rename every variable.

Reply:
Good to go with 68d25c3a04e6f03942e492d7ca65cbe4c9e897e9